collect_app/src/main/java/org/odk/collect/android/backgroundwork/BackgroundWorkUtils.java

```diff
@@ -15,13 +16,13 @@ private BackgroundWorkUtils() {
     }
 
     public static long getPeriodInMilliseconds(String period) {
-        if (period.equals(Collect.getInstance().getString(R.string.every_one_hour_value))) {
+        if (period.equals(TranslationHandler.getString(Collect.getInstance(), R.string.every_one_hour_value))) {
```

Comment:
This method could take a `Context` param in our just be inlined it looks like as it's only called from one place that has access to `Application`.

Reply:
Done.